include/HUDL/HUDL.hpp

```diff
@@ -105,8 +105,8 @@ class HUDL {
     uint32_t thermTemps[4] = {};
 
     static constexpr uint16_t OBJECT_DICTIONARY_SIZE = 30;
-    static constexpr uintptr_t TMS_TPDO0_COB_ID = CO_COBID_TPDO_DEFAULT(0);
-    static constexpr uintptr_t TMS_TPDO1_COB_ID = CO_COBID_TPDO_DEFAULT(1);
+    static constexpr uintptr_t TMS_NODE_ID = 0x02;
```

Comment:
The TMS's node ID is 0x08. You can find all the node IDs that the boards are supposed to have in [this doc](https://docs.google.com/spreadsheets/d/1KUt-7LYNFojA4RDHoGJRBh8pZ9A7QXzuhNFz2hbOP20/edit?usp=sharing), which is also pinned in #subgroup-firmware with the oh so helpful label "Can whatever"